LabOne.tex

```diff
@@ -0,0 +1,21 @@
+\chapter{Bookstore UML Modeling \\
+  \small{\textit{-- Evan Ciok, Sophia DiCuffa, Carson McManus}}
+  \index{Chapter!introduction}
+  \index{Lab One}
+  \label{Chapter::LabOne}}
+
+% Add a section and label it so that we can reference it later
+\section{Bookstore UML Modeling \label{Section::LabOne}}
+
+\begin{figure}
+    \centering
+    \scalebox{0.8}{\includegraphics{Figures/bookstore/usecase.jpg}}
+    \caption{\label{Figure::useCase} A use-case diagram for the e-Bookstore system that supports the list of functions shown above.}
+    \end{figure}
+
+    \begin{figure}
+        \centering
+        \scalebox{0.8}{\includegraphics{Figures/bookstore/uml.png}}
+        \caption{\label{Figure::useCase} A class diagram for the e-Bookstore system that illustrates the choices of the main classes and 
+        appropriate relationships among them.}
+        \end{figure}
```

Comment:
nit: atrocious formatting lol

Reply:
i cant see it on my end so idk how to fix :/ i just copied the prof's example

Reply:
I was talking about the litteral whitespace indentation in the latex code, not how it renders in the pdf.

Reply:
ohh my b